<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICMachineFigure</class>
 <widget class="QWidget" name="ICMachineFigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMessageBox QAbstractButton
{
    min-width: 90px;
	min-height:30px;
}
</string>
  </property>
  <widget class="QLabel" name="figure">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>751</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/resource/machine_struct.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="xsec1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="xsec2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="xsec3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="xsec4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="xmax">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="xmin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
